v21.1/information-schema.md

```diff
@@ -294,6 +510,18 @@ Column | Description
 `table_type` | Type of the table: `BASE TABLE` for a normal table, `VIEW` for a view, or `SYSTEM VIEW` for a view created by CockroachDB.
 `version` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table. Note that this column is an experimental feature used for internal purposes inside CockroachDB and its definition is subject to change without notice.
 
+### type_privileges
+
+`type_privileges` contains information about privileges on the user-defined types in the current database.
+
+Column | Description
+-------|-----------
+`grantee` | Username of user with privilege grant.
+`type_catalog` | Name of the database that contains the type (always the current database).
+`type_schema` | Name of the schema that contains the type (always `pg_catalog`).
```

Comment:
this is not always pg_catalog. for builtin types it's pg_catalog; for user-defined types it's public

also, i think this table is in 20.2 as well

Reply:
actually, for UDTs the schema could be whichever schema it was created in